src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/CodeView/CodeViewNative.cs

```diff
@@ -0,0 +1,2364 @@
+            CV_ARM64_X13 = 63,
+            CV_ARM64_X14 = 64,
+            CV_ARM64_X15 = 65,
+            CV_ARM64_IP0 = 66,
+            CV_ARM64_IP1 = 67,
+            CV_ARM64_X18 = 68,
+            CV_ARM64_X19 = 69,
+            CV_ARM64_X20 = 70,
+            CV_ARM64_X21 = 71,
+            CV_ARM64_X22 = 72,
+            CV_ARM64_X23 = 73,
+            CV_ARM64_X24 = 74,
+            CV_ARM64_X25 = 75,
+            CV_ARM64_X26 = 76,
+            CV_ARM64_X27 = 77,
+            CV_ARM64_X28 = 78,
+            CV_ARM64_FP = 79,
+            CV_ARM64_LR = 80,
+            CV_ARM64_SP = 81,
+            CV_ARM64_ZR = 82,
+            CV_ARM64_PC = 83,
+
+            // status registers
+            CV_ARM64_NZCV = 90,
+            CV_ARM64_CPSR = 91,
+
+            // 32-bit floating point registers
+            CV_ARM64_S0 = 100,
+            CV_ARM64_S1 = 101,
+            CV_ARM64_S2 = 102,
+            CV_ARM64_S3 = 103,
+            CV_ARM64_S4 = 104,
+            CV_ARM64_S5 = 105,
+            CV_ARM64_S6 = 106,
+            CV_ARM64_S7 = 107,
+            CV_ARM64_S8 = 108,
+            CV_ARM64_S9 = 109,
+            CV_ARM64_S10 = 110,
+            CV_ARM64_S11 = 111,
+            CV_ARM64_S12 = 112,
+            CV_ARM64_S13 = 113,
+            CV_ARM64_S14 = 114,
+            CV_ARM64_S15 = 115,
+            CV_ARM64_S16 = 116,
+            CV_ARM64_S17 = 117,
+            CV_ARM64_S18 = 118,
+            CV_ARM64_S19 = 119,
+            CV_ARM64_S20 = 120,
+            CV_ARM64_S21 = 121,
+            CV_ARM64_S22 = 122,
+            CV_ARM64_S23 = 123,
+            CV_ARM64_S24 = 124,
+            CV_ARM64_S25 = 125,
+            CV_ARM64_S26 = 126,
+            CV_ARM64_S27 = 127,
+            CV_ARM64_S28 = 128,
+            CV_ARM64_S29 = 129,
+            CV_ARM64_S30 = 130,
+            CV_ARM64_S31 = 131,
+
+            // 64-bit floating point registers
+            CV_ARM64_D0 = 140,
+            CV_ARM64_D1 = 141,
+            CV_ARM64_D2 = 142,
+            CV_ARM64_D3 = 143,
+            CV_ARM64_D4 = 144,
+            CV_ARM64_D5 = 145,
+            CV_ARM64_D6 = 146,
+            CV_ARM64_D7 = 147,
+            CV_ARM64_D8 = 148,
+            CV_ARM64_D9 = 149,
+            CV_ARM64_D10 = 150,
+            CV_ARM64_D11 = 151,
+            CV_ARM64_D12 = 152,
+            CV_ARM64_D13 = 153,
+            CV_ARM64_D14 = 154,
+            CV_ARM64_D15 = 155,
+            CV_ARM64_D16 = 156,
+            CV_ARM64_D17 = 157,
+            CV_ARM64_D18 = 158,
+            CV_ARM64_D19 = 159,
+            CV_ARM64_D20 = 160,
+            CV_ARM64_D21 = 161,
+            CV_ARM64_D22 = 162,
+            CV_ARM64_D23 = 163,
+            CV_ARM64_D24 = 164,
+            CV_ARM64_D25 = 165,
+            CV_ARM64_D26 = 166,
+            CV_ARM64_D27 = 167,
+            CV_ARM64_D28 = 168,
+            CV_ARM64_D29 = 169,
+            CV_ARM64_D30 = 170,
+            CV_ARM64_D31 = 171,
+
+            // 128-bit SIMD registers
```

Comment:
Just a note: Looks like CodeView doesn't support SVE scalable and predicate registers yet